content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -635,6 +635,17 @@ or updating objects that contain Pod templates, such as Deployments, Jobs, State
 See [Enforcing Pod Security at the Namespace Level](/docs/concepts/security/pod-security-admission)
 for more information.
 
+### rbac.authorization.kubernetes.io/autoupdate
+
+Example: `rbac.authorization.kubernetes.io/autoupdate: "false"`
+
+Used on: ClusterRole, ClusterRoleBinding, Role, RoleBinding
+
+When this annotation is set to `true`, default RBAC ClusterRole and ClusterRoleBinding objects are automatically updated at server start to add missing permissions and subjects (extra permissions and subjects are left in place). To prevent autoupdating a particular role or rolebinding, set this annotation to `false`.
+If you create your own ClusterRole and set this annotation, `kubectl auth reconcile`
```

Comment:
```suggestion
If you create your own RBAC objects and set this annotation to `false`, `kubectl auth reconcile`
```

Reply:
(nit) we'd typically prefer to write ”false” without backticks - see https://kubernetes.io/docs/contribute/style/style-guide/#use-normal-style-for-string-and-integer-field-values

However, to make it clear that this is actually the **string** `"false"`, try:
```suggestion
If you create your own RBAC objects and set this annotation to `"false"`, `kubectl auth reconcile`
```